### ISSUE TITLE
Model for 12 class classification

### ISSUE THREAD

**Issue by Asha2411** [jgmeyerucsd/drug-class#13]
Would it be possible for you to please share the model for 12 class classification?
Thank you in advance!

**Comment by agitter** [comment]
@jgmeyerucsd do you have this saved model available?

**Comment by jgmeyerucsd** [comment]
I can check on my hard drive at home tonight

**Comment by jgmeyerucsd** [comment]
@Asha2411  which model in particular are you looking for?? I believe we reported results from the following:
12 class RF
12 class CNN
12-class CNN multi-class

**Comment by Asha2411** [comment]
Thank you for your reply. I'm looking for 12 class CNN single class model.

**Comment by jgmeyerucsd** [comment]
@Asha2411   I'm afraid I don't have the single class models but I found the 12-class multi-output model. This used fold 5 as the validation set. Can you please let me know if this works for you? It is pytorch model. https://drive.google.com/file/d/1-xRnNk7tnzsmsZymaXBedR6uoLn___Qp/view?usp=sharing

If it works for you I will add a permanent link to a version on zenodo.

**Comment by Asha2411** [comment]
No. I'm sorry, I was looking for 12 class single class model. Anyways thank you for your help.

**Comment by jgmeyerucsd** [comment]
@Asha2411 would you be interested in a similar 12-class model (not the exact model used in the paper)? 

You could try training with this notebook:
https://github.com/jgmeyerucsd/drug-class/blob/master/IMG_CNN/JCIM_revision_large_data_unfrozen.ipynb

The environment setup is challenging because it requires fastai v0.7; there is a link to the install instructions at the top of the notebook. 

Alternatively, I could try to train the model myself in a few weeks after some deadlines have passed. 

Please let me know if any of these options are of interest to you. I am happy to help if I can.

**Comment by Asha2411** [comment]
The problem is fastai v0.7 isn't available for windows. So, now I gave up on this project. Sorry!
 
I was thinking to take your Pytorch model and covert it into tf and then train. But it seems difficult now
